pages/docs/examples/email-sequence.mdx

```diff
@@ -0,0 +1,156 @@
+import { ResourceGrid, Resource } from 'src/shared/Docs/Resources';
+import { Example } from 'src/shared/Docs/Examples';
+
+import { EnvelopeIcon } from "@heroicons/react/24/outline";
+
+
+export const description = "See how to implement an email sequence with Inngest"
+
+# Email Sequence
+
+A drip campaign is usually based on your user's behavior.
+
+Let's say you want to create the following campaign:
+
+- Send every user a welcome email when they join.
+  - If a user received an email: wait a day and then follow-up with pro user tips meant for highly engaged users.
+  - Otherwise: wait for up to three days and then send them the default trial offer, but only if the user hasn't already upgraded their plan in the meantime.
+
+This page provides an overview on how to use Inngest to build reliable marketing campaigns, as well as all the related materials to this feature.
+
+## Quick Snippet
+
+Below is an example of how such a campaign would look like:
+
+```javascript
+const signupDripCampaign = inngest.createFunction(
+  { id: "signup-drip-campaign" },
+  { event: "app/signup.completed" },
+  async ({ event, step }) => {
+    const { user } = event.data;
+    const { email, first_name } = user
+    const welcome = "Welcome to ACME";
+
+    const { id: emailId } = await step.run("welcome-email", async () => {
+      return await sendEmail(
+        email,
+        welcome,
+        <div>
+          <h1>Welcome to ACME, {user.firstName}</h1>
+        </div>
```

Comment:
the inline JSX may be a little distracting to take away from the concepts, but it's ok for now!